Screen recording

watermelonpizza opened the pull request.

aeh commented:
May need to drop node 10 tests so that they pass or add the `catch (error) {}` back in again.

watermelonpizza commented:
> May need to drop node 10 tests so that they pass or add the `catch (error) {}` back in again.

Removed nodejs versions `8` and `10` and added in `14`

aeh commented:
LGTM
